doc/source/index.rst

```diff
@@ -101,6 +106,35 @@ The astronomy module enables computation of certain parameters of interest for s
     >>> astronomy.sun_zenith_angle(utc_time, lon, lat)
     62.685986438071602
 
+
+.. envvar:: PYORBITAL_CONFIG_PATH
+
+   It is possible (but not mandatory) to define this environment variable to
+   have full control of certain static data used by Pyorbital:
+
+   Pyorbital comes with a file platforms.txt that maps a satellite name to the
+   NORAD identifier. This internal file is accessed by Pyorbital without the
+   user having to do anything.  But if you need to change or update this file
+   you can make your own copy and place in the directory pointed to by this
+   environment variable.
```

Comment:
Does the file still need to be called `platforms.txt`? Why is this a directory and not a single file path?

Reply:
Well, I think you are right that it could be a path to a single file, but way back I think we thought there would be more config files needed that could be accessed via this directory. Didn't want to change this now. We could make an issue perhaps?